src/fetch/rewards.py

```diff
@@ -0,0 +1,170 @@
+"""Functionality for computing solver rewards
+
+The only function used in other parts of the code is `compute_solver_rewards` for generating
+aggregate reward information per solver."""
+
+from pandas import DataFrame, Series
+from src.fetch.prices import TokenConversion
+
+EPSILON_UPPER = 12000000000000000
+EPSILON_LOWER = 10000000000000000
```

Comment:
Python supports underscored integers...
```py
>>> print(1000 == 1_000)
True
```

```suggestion
EPSILON_UPPER = 12_000_000_000_000_000
EPSILON_LOWER = 10_000_000_000_000_000
```

Reply:
Nice. I am using this all the time in julia. But did not know it works in python as well.